core/storage/job/gae_models.py

```diff
@@ -86,6 +93,15 @@ def is_cancelable(self):
 
     @classmethod
     def get_recent_jobs(cls, limit, recency_msec):
+        """Gets no more than a number of jobs within a given recency.
+
+        Args:
+            limit: int. A limit on the number of jobs to return.
+            recency_msec: int. How recent of a job to search for.
+
+        Returns:
+            A list of jobs. Possiably none.
```

Comment:
`list(...) or None. A list of all jobs that ...`

(where you'll have to fill in the '...' bits)